### DISCUSSION TITLE
Orcaslicer 100% failure after update to 2.0

### DISCUSSION THREAD

**Discussion by darkuncle33** [OrcaSlicer/OrcaSlicer#4839] [General]
I am having an issue with first layers. Any skirts or brims as well as first layer walls print fine, but as soon as it starts printing solid infill it prints too high. After that nothing touches the build plate. I looked at the gcode and before it starts the bottom solid infill it at g1 z.2 but still to high.
Super slicer works fine and old gcode from orcaslicer still prints fine.
Sorry I am not good at explaining things like this.
The first pic is Superslicer
The next two are orcaslicer
![IMG_20240401_180551 1](https://github.com/SoftFever/OrcaSlicer/assets/79282920/d2a4cf30-8bc5-4120-9b62-33a755a1f266)
![IMG_20240401_181821 1](https://github.com/SoftFever/OrcaSlicer/assets/79282920/385ff5ce-d64d-4847-8b77-eb903d5ec870)
![IMG_20240401_181843 1](https://github.com/SoftFever/OrcaSlicer/assets/79282920/a507ebee-e1c0-4bf2-bf8f-b64c4bf1b024)

**Comment by TexZeTech** [comment]
Cannot say I've had that issue.

What I can say this is too little information.

OS information:
OrcaSlicer Version:
Printer Make/Model:
Firmware version:

.3MF project so we can open it up on our end. note: You have to put it in a .zip file.

**Reply by darkuncle33** [reply]
Windows 10
orcaslicer 2.0
voron trident
Klipper

I am using orcaslicer 2.0 / same pc same install on my voron 2.4 and it works fine. 
Only haveing trouble with the Trident.
Gcode that I made 4 days ago with orcaslicer 1.6 still works fine on the trident. when i slice the same model again with 2.0 I have the same issue.
The Trident is useing Voron Tap and everything I slice with superslicer works.
Only orcaslicer 2.0 has the issue.
I looked at the print settings and they are correct as far as bottom layers. It does the same thing with other setting profiles. Only on the tridint and only on 2.0.
I sliced some gcode on another pc with orcaslicer 2.0 and it still fails.

**Reply by TexZeTech** [reply]
.3MF project so we can open it up on our end. note: You have to put it in a .zip file.

**Comment by darkuncle33** [comment]
[bezel.zip](https://github.com/SoftFever/OrcaSlicer/files/14844581/bezel.zip)

**Reply by TexZeTech** [reply]
Can I see the start code in superslicer? I know mine in Orca is 



`PRINT_START EXTRUDER=[first_layer_temperature] BED=[first_layer_bed_temperature]`
edit: I lied had wrong start code listed.

Then I run everything from the printer macro wise.


As soon as I'm done a print I will be trying to run your .3MF as is with only editing the start code block to mach mine.

**Reply by TexZeTech** [reply]
Just started printing and this is how it looks.
<img src="https://github.com/SoftFever/OrcaSlicer/assets/7254540/27a0eb25-fbcd-462d-a953-2968a46bffd9" width=80% height=80%>

What I would recommend trying depending on how you have your start macro is as follows


```
NOZZLE_WIPE
PRINT_START EXTRUDER=[first_layer_temperature] BED=[first_layer_bed_temperature]
Voron_Purge

```

But if you are like me I would just add the Nozzle_wipe macro and the Voron_purge macro right into your startup script like this.

[PRINT_START.txt](https://github.com/SoftFever/OrcaSlicer/files/14845305/PRINT_START.txt)


Granted the above is all assuming your start macro is similar to mine.

**Reply by TexZeTech** [reply]
[TexZeTech_Body2_ABS_9m19s.zip](https://github.com/SoftFever/OrcaSlicer/files/14845331/TexZeTech_Body2_ABS_9m19s.zip)

This has the start code changed to match 

```PRINT_START EXTRUDER=[first_layer_temperature] BED=[first_layer_bed_temperature]```
